spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StandardizationJob.scala

```diff
@@ -206,7 +176,15 @@ object StandardizationJob {
   }
 
   private def getGenericOptions(cmd: StdCmdConfig): HashMap[String,Option[RawFormatParameter]] = {
-    HashMap("charset" -> cmd.charset.map(StringParameter))
+    val mode = if (cmd.failOnInputNotPerSchema) {
+      "FAILFAST"
```

Comment:
Would these values maybe deserve to be referenced from a named value? (string, enum value, ...)

Reply:
Well, it could be used/shared in the future, but this is more of "_things being of a correct type_" kind of comment - it naively seems that if a `val` holds a mode that could only have certain states, it naturally calls for an enum-like type.

I understand that it is not a very important point anyway, so you may very fell decide to not to reflect it. 😄